media/doc/Romanian translation notes.txt

```diff
@@ -35,14 +35,12 @@ La acestea se adaugă și câteva norme (deduse) din „greșeli frecvente” ra
 
 În resursele românești din ReactOS există și o serie de particularizări care fie nu se regăsesc în alte traduceri fie din cauza diferențelor de natură ale proiectelor, fie e vorba de termeni încă disputați sau neadoptați în localizările altor proiecte din diverse motive. Multe dintre acestea au fost menționate/discutate în grupul „Diacritice” [4]. Câteva dintre cele mai relevante:
  15. În glosar sunt preferate:
-  - „logică computațională” vs. „software”
-  - „configurare”/„particularizare” vs. „Setare”
-  - „modul pilot” (sau doar „pilot”) vs. „driver”
+  - „Confirmă”/„Anulează” vs. „OK”/„Renunță”
  16. Acceleratorii (combinațiile de taste de gen „«Ctrl» + «tastă»”) rămân aceiași ca în limba engleză.
  17. Tastele de acces (combinațiile de taste de gen „«Alt» + «tastă»”) nu rămân neapărat la fel ca în limba engleză. În mulțimea de taste cu această utilizare:
   - Este recomandată evitarea utilizării literelor „s” și „t” pe post de taste de acces pentru a preveni confuzia între acestea și diacriticele „ș” și „ț”.
   - „f” și „n” sunt rezervate pentru „Confirmă” și respectiv „Anulează”; „a” pentru „Aplică” (dacă există în aceiași fereastră). În cazul ferestrelor de tip asistent, „a”, „o”, „f” și „n” sunt rezervate respectiv pentru „Înainte”, „Înapoi”, „Sfârșit” și „Anulează”. [NOTĂ]
-  - În cazul interfețelor grafice opțiunilor „Da” și „Nu” le corespund tastele de acces „a” și „u”. (În cazul interfețelor linie-de-comandă, corespunzătoare opțiunilor „Da|Nu”, care NU sunt taste de acces, rămân „d” și „n”.)
+  - În cazul interfețelor grafice opțiunilor „Da” și „Nu” le corespund tastele de acces „a” și „u”.
```

Comment:
The line you just changed was specifying that for program queries towards the user, the keys for "yes" and "no” in Romanian translated resources were "d" and "n". To have this stated explicitly across ReactOS **is useful**. This did not intended to be a guide on how the options "Da" and "Nu" were to be presented to the user, as I suspect was the reason behind your change (judging from [one of your comments](https://github.com/reactos/reactos/pull/5939/files#r1389256032)).

Reply:
I understand. I'll revert this change, but with slash instead of pipe.

Reply:
I changed it, so now I re-added the text with slash instead of pipe. I hope it's good now.

Reply:
I changed it now to Use slash instead of pipe. Is it good?

Reply:
> I changed it now to Use slash instead of pipe. Is it good?

Yes, that's better.